src/screens/app/screens/AddTask/screens/components/TaskDifficulty.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react'
+
+import { CardButton } from 'library/components/CardButton'
+import { Point } from 'library/components/Point'
+
+interface TaskDifficultyProps {
+  taskDifficulty: TaskDifficulty
+  taskDifficultyIndex: number
+  color: string
+  selectedIndex: number
+  onAction: (taskDifficultyIndex: number) => void
+  isFirstItem: boolean
+  isLastItem: boolean
+}
+
+export const TaskDifficulty: FC<TaskDifficultyProps> = ({
+  taskDifficulty,
+  taskDifficultyIndex,
+  color,
+  selectedIndex,
+  onAction,
+  isFirstItem,
+  isLastItem,
+}) => {
+  const DIFFICULTY_NAME_MAPPING = {
```

Comment:
translation will come after ?

Reply:
I will yes, I need to ask the translations